fvtest/compilertriltest/IfxcmpgeReductionTest.cpp

```diff
@@ -409,15 +413,15 @@ TEST_P(UInt64ReductionTest, Reduction)
          "    (iflucmpge target=\"t\""
          "      (land"
          "        (lload parm=0)"
-         "        (lconst %s) )"
-         "      (lconst %s) ) )"
+         "        (lconst %llu) )"
+         "      (lconst %llu) ) )"
          "  (block name=\"f\""
          "    (ireturn (iconst 0) ) )"
          "  (block name=\"t\""
          "    (ireturn (iconst 1) ) ) )",
 
-         std::to_string(param.val2).c_str(), // value of and const
```

Comment:
@fjeremic  I'm not quite sure what this comment (and the one below it) are trying to say.

Reply:
This is the value that will be supplied to the `%llu` format specifier for the `lconst` child of the `land`.  It took me a couple of passes to parse it too.  I think calling out the IL opcodes explicitly would make this clearer.  e.g.,
```
// value of 'lconst' for 'land'
```

Reply:
@0xdaryl  thanks. And yeah, single quotes always help : ]